src/Core/Framework/DataAbstractionLayer/Dbal/EntityAggregator.php

```diff
@@ -137,7 +137,7 @@ private function fetchAggregation(Aggregation $aggregation, EntityDefinition $de
             $clone->setTerm(null);
         }
 
-        $scoreCritera = clone $clone;
+        $scoreCriteria = clone $clone;
```

Comment:
Thanks for good catch, @wannevancamp  

I'm rather greedy :) We have some similar in other places. You don't mind if you help to fix it as well in a single MR?

Thanks in advance.

Reply:
@trandangtri I've fixed the typos in the other files too.